base/Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM buildpack-deps:focal
 COPY install-packages /usr/bin
 
 ### base ###
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
⚠️ `ENV` variables carry over into Gitpod workspaces. Maybe it would be surprising to users if the Debian frontend is permanently non-interactive (even in their interactive terminals)?

I typically set this inline for each `apt-get install` command to work around this. Alternatively, we could unset it again at the end of `base/Dockerfile` or `full/Dockerfile`.

EDIT: Is this actually even needed here? `install-packages` already has it locally.

Reply:
ARG means this is valid only during docker build

Reply:
Today I learned:

> Unlike an `ARG` instruction, `ENV` values are always persisted in the built image.